src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -53,6 +56,31 @@ const CompleteIncident = props => {
     setEditing(!editing);
   };
 
+  // for incident deletion button
+  const { confirm } = Modal;
+
+  function toggleDelete() {
+    confirm({
+      title: `Do you want to DELETE incident #${incident.incident_id}?`,
+      icon: <ExclamationCircleOutlined />,
+      content: 'Once you click OK this dialogue will disappear in one second',
+      onOk() {
+        oktaAxios
+          .delete(`/dashboard/incidents/${incident.incident_id}`)
+          .then(res => {
+            window.location.reload();
+          })
+          .catch(err => {
+            console.log(err);
+          });
+        return new Promise((resolve, reject) => {
+          setTimeout(Math.random() > 0.5 ? resolve : reject, 1000);
```

Comment:
Why does this promise reject based on a random number?

Reply:
To keep you on your toes.